src/styles/core/colors/_dark.scss

```diff
@@ -88,4 +88,5 @@
   --color-navigator-item-hover: #{change-color(dark-color(fill-blue), $alpha: 0.5)};
 
   --color-card-shadow: #{change-color(light-color(fill), $alpha: 0.04)};
+  --color-link-block-card-border: #{change-color(light-color(fill), $alpha: 0.25)};
```

Comment:
Is the alpha value intentionally higher here?

Reply:
yep, per Sam specs.